.circleci/config.yml

```
@@ -718,9 +718,9 @@ jobs:
                 value: << pipeline.git.branch >>
           steps:
             - aws-cli/setup:
-                aws_access_key_id: AWS_ACCESS_KEY_ID_ARTIFACTS
-                region: AWS_REGION_ARTIFACTS
-                aws_secret_access_key: AWS_SECRET_ACCESS_KEY_ARTIFACTS
+                aws_access_key_id: $AWS_ACCESS_KEY_ID_ARTIFACTS
```

Would it be correct?

```suggestion
                aws_access_key_id: ${AWS_ACCESS_KEY_ID_ARTIFACTS}
```

---

It shouldn't matter indeed, but they messed this up on a minor and had to improvise to fix this in a forward-compatible way. It's likely they only considered `${VAR_NAME}` as a format in their fix. Testing that hypothesys in https://github.com/mui/material-ui/pull/38956